public/locales/en-US/translations.json

```diff
@@ -386,6 +386,12 @@
   "validator_history.date": "Date",
   "validator_history.missed": "Missed",
   "validator_history.score": "Score",
+  "seller": "Seller",
+  "buyer": "Buyer",
+  "offerer": "Offerer",
+  "token_taxon": "Token Taxon",
+  "uri": "URI",
+  "owner_label": "Owner",
```

Comment:
Can you replace the original usages of `owner` -> `Owner's Account` with something like `owner_account`? Then have `owner` -> "Owner"

Otherwise I think the `owner_label` is a bit of a confusing name that other folks won't know to search for sinc emost of the other names for things in the translations are far more direct. (If it would change 5+ files, probably not worth it, but if it's just a couple places I think the consistency would be worth it)

Reply:
I don't even see owner -> Owner's Account used anywhere.. guess I can change it? @ckniffen